build/containers/deconstructsig/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM debian:buster-slim
+
+ENV LC_ALL=C.UTF-8
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        python3-pandas python3-click r-bioc-bsgenome r-cran-gplots procps && \
+    rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN echo "install.packages(\"BiocManager\", repos=\"https://cran.r-project.org\")" | R --no-save && \
+    echo "BiocManager::install(\"BSgenome.Hsapiens.UCSC.hg19\")" | R --no-save && \
+    echo "BiocManager::install(\"BSgenome.Hsapiens.UCSC.hg38\")" | R --no-save && \
+    echo "install.packages(\"deconstructSigs\", repos=\"https://cran.r-project.org\")" | R --no-save
```

Actually this container does not get built properly because this step returns the following in the workflow logs:
```
#8 91.49 Warning message:
#8 91.49 package ‘deconstructSigs’ is not available (for R version 3.5.2) 
#8 DONE 91.5s
```

So the method is not installed, in the end. I digged a bit and it's because the repo was removed from cran.r. This should be fixed by installing the package from the archive repo [deconstructSigs](https://cran.r-project.org/src/contrib/Archive/deconstructSigs/deconstructSigs_1.8.0.tar.gz) 

```suggestion
    echo "install.packages(\"https://cran.r-project.org/src/contrib/Archive/deconstructSigs/deconstructSigs_1.8.0.tar.gz\", repos=NULL, type=\"source\")" | R --no-save
```